includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -1122,6 +1138,48 @@ private function check_attr_spec_rule_value_regex_casei( $node, $attr_name, $att
 		return AMP_Rule_Spec::NOT_APPLICABLE;
 	}
 
+	/**
+	 * Check if attribute has a valid host value
+	 *
+	 * @since 0.7
+	 *
+	 * @param DOMElement       $node           Node.
+	 * @param string           $attr_name      Attribute name.
+	 * @param array[]|string[] $attr_spec_rule Attribute spec rule.
+	 *
+	 * @return string:
+	 *      - AMP_Rule_Spec::PASS - $attr_name has a value that matches the rule.
+	 *      - AMP_Rule_Spec::FAIL - $attr_name has a value that does *not* match rule.
+	 *      - AMP_Rule_Spec::NOT_APPLICABLE - $attr_name does not exist or there
+	 *                                        is no rule for this attribute.
+	 */
+	private function check_attr_spec_rule_valid_url( $node, $attr_name, $attr_spec_rule ) {
+		if ( isset( $attr_spec_rule[ AMP_Rule_Spec::VALUE_URL ] ) ) {
+			if ( $node->hasAttribute( $attr_name ) ) {
+				$attr_value   = $node->getAttribute( $attr_name );
+				$urls_to_test = preg_split( '/\s*,\s*/', $attr_value );
```

Comment:
Since only `srcset` has comma-separated list of URLs, maybe this should check of `'srcset' === $attr_name` for whether to split. Maybe that's overkill. My only concern would be if a non-`srcset` URL happening to have a comma in it and that this would result in the attribute being invalid. 

The alternative could be to do this: 

```php
$urls_to_test = preg_split( '#\s*,\s*(?=https?://)#', $attr_value );
```

That will ensure that it will only split where a comma is followed by another URL.